### DISCUSSION TITLE
Rename or Remove Existing Backpacks

### DISCUSSION THREAD

**Discussion by Zsynh** [P3pp3rF1y/SophisticatedBackpacks#949] [Feature / Idea Archive]
Howdy, Team -

I'm hoping you might be open to adding a means to either rename existing packs in the game to something more user-friendly or allow a way to remove one (or all) from the database. This would really help me out as a server administrator - if someone has lost their pack and makes a new one, they'll still have two pack entries in the database even though only one is actually in the game. Lather, rinse, repeat a few times and we have players with a dozen entries and I don't think there's any way to tell which is most recent or where it was last seen. We only have a few players, so it wouldn't be a huge burden for me to rename them at the command line if non-op players wouldn't be able to.

Thanks for reading, and for a really good mod.

**Comment by P3pp3rF1y** [comment]
There's a way to say which one was more recently used just do `/sbp list [player name]` and it will list them ordered by how recently they were accessed by the given player.

And regarding removing the data to be honest I don't understand the motivation to do that - if the aim is to save data on the disk then I would argue that deleting chunks of the world can save you way more than that as a single additional chunk likely takes way more on the disk than all the lost backpacks combined

**Reply by Zsynh** [reply]
Thanks for the quick reply! The list command should help. In terms of removing data, the motivation is simply ease of figuring out what's what. I personally have 4 packs I use all at the same time, and can't tell which is which easily especially given there are 8 or 9 I lost along the way that still show up in my 'give' list. I'd just like to delete the packs that aren't valid any longer.
